package.json

```diff
@@ -43,7 +43,7 @@
   },
   "scripts": {
     "start": "npm run check-env && npm run dev",
-    "dev": "webpack-dev-server --config webpack.dev.js --host 0.0.0.0",
+    "dev": "webpack-dev-server --config webpack.dev.js",
```

Comment:
Why you do dis?

Reply:
Does the frontend still run from docker?

Reply:
afaik it doesn't (or at least I am not developing with docker). is this required for docker work?

Reply:
oh and btw I did it because windows doesn't resolve 0.0.0.0 and it was annoying me ;)